Add AI section to Case Management docs (#37356)
DataDog/documentation: content/en/incident_response/case_management/ai/_index.md

---

> title: AI

Reviewer: Update the title and add a description for AI retrieval on isolated context
I recommend folding in the MCP section into the main AI index page.

```suggestion
title: AI Tools for Case Management
description: Datadog Case Management integrates with AI tools to help automate case triage, assignment, and resolution using the MCP Server and custom agents.
aliases:
- /service_management/case_management/mcp_server/
- /incident_response/case_management/mcp_server/
```

---

> Datadog Case Management integrates with AI tools to support different workflows:

Reviewer: Add more context on what AI in Case Management does for customer use cases:

```suggestion
Datadog Case Management is where you track and assign work to AI agents the same way you do for people. It integrates with the Datadog MCP Server and custom agents built with Bits Agent Builder to support different workflows.
```

- **MCP Server**: Use the Case Management toolset from the Datadog MCP Server to interact with cases from your preferred AI-powered client.

> - **Custom Agents**: Assign cases to specialized agents built with Bits Agent Builder to automate the full case lifecycle, from initial triage to follow-up and resolution.

Reviewer: Recommend folding in the MCP server in the index page with a short introduction of the Custom Agent that references out to the Custom Agent docs:

```suggestion
## MCP Server

The Datadog MCP Server exposes a `cases` toolset so that AI agents that support the [Model Context Protocol (MCP)][2] can access Case Management data. The `cases` toolset allows AI agents to create, search, update, and manage cases. Supported workflows include:

- **Searching for cases** based on status, priority, project, or other filters
- **Retrieving details of a case** to understand the latest timeline of actions and remaining work
- **Creating a new case** to track information related to an ongoing investigation
- **Updating an existing case** with new findings, links to related Jira tickets, or escalated priority

For setup instructions and full details on the `cases` toolset, see the [Datadog MCP Server documentation][1].

## Custom agents

{{< callout url="https://www.datadoghq.com/product-preview/custom-agents-in-case-management/" btn_hidden="false" header="Join the Preview">}} Case Management integration with custom agents is in Preview.{{< /callout >}}

Assign cases to specialized agents built with [Bits Agent Builder][3] to automate the full case lifecycle, from initial triage to follow-up and resolution. For example use cases, agent archetypes, and manual and automated assignment, see [Custom Agents][4].
```

> {{< callout url="https://www.datadoghq.com/product-preview/custom-agents-in-case-management/" btn_hidden="false" header="Join the Preview!">}} Case Management integration with custom agents is in Preview.{{< /callout >}}

Reviewer: ```suggestion
```

## Features
{{< whatsnext desc="Learn more:">}}
    {{< nextlink href="/incident_response/case_management/ai/mcp_server" >}}MCP Server{{< /nextlink >}}
    {{< nextlink href="/incident_response/case_management/ai/custom_agents" >}}Custom Agents{{< /nextlink >}}

> {{< /whatsnext >}}

Reviewer: ```suggestion
[1]: /mcp_server
[2]: https://modelcontextprotocol.io/
[3]: /actions/agents/
[4]: /incident_response/case_management/ai/custom_agents/
```
